### ISSUE TITLE
Add an vr space mover

### ISSUE THREAD

**Issue by boomtime6** [emmVRC/emmVRC-Public#1615]
Add an vr space mover.

**Comment by rakosi2** [comment]
Like [OVR Advanced Settings](https://store.steampowered.com/app/1009850/OVR_Advanced_Settings/)?

**Comment by MintLily** [comment]
I good video to learn how to set up [OVR Advanced Settings](https://store.steampowered.com/app/1009850/OVR_Advanced_Settings/) is here: https://youtu.be/E4ZByfPWTuM
Implementing this into the mod is rather pointless if OVRAS is already great to use as is.

**Comment by boomtime6** [comment]
I have already used this or advanced settings, it is very inconvenient in terms of management, vr space mover was in one of the clients, and it had good management.

**Comment by rakosi2** [comment]
The only mod that does it is [OculusPlayspaceMover](https://github.com/Rafacasari/Playspace-Mover/) which only works with the Oculus version of VRChat

**Comment by boomtime6** [comment]
I use this function in client,but when come update 1149 its stop working on my htc vive pro

**Comment by MintLily** [comment]
TLDR; We are not implementing this into the mod. Use [OVR Advanced Settings](https://store.steampowered.com/app/1009850/OVR_Advanced_Settings/) and use the [YouTube video](https://youtu.be/E4ZByfPWTuM) if you need help.
